README.md

```diff
@@ -1,117 +1,131 @@
 
-Redis Enterprise    -   `redislabs/redis:5.4.14-31` or `redislabs/redis:5.4.14-31.rhel7-openshift`
 
-Operator            -   `redislabs/operator:5.4.14-7` or `redislabs/operator:5.4.14-7.rhel7`
+### Installation
+The "Basic" installation deploys the operator (from the current release) with the default Ubuntu/Alpine base OS images from DockerHub and default settings.
+This is the fastest way to get up and running with a new Redis Enterprise on Kubernetes.
 
-Services Rigger     -   `redislabs/k8s-controller:5.4.14-7` or `redislabs/k8s-controller:5.4.14-7.rhel7`
+1. Create a new namespace:
 
-Service Broker      -   `redislabs/service-broker:78_4b9b17f` or `redislabs/service-broker:78_4b9b17f.rhel7`
+    ```bash
+    kubectl create namespace demo
+    ```
 
-## Basic installation
-The basic installations deploys the operator from the current release with the default Ubuntu/Alpine base OS images from DockerHub and default settings.
-This is the fastest way to get up and running with a new cluster in most environments.
-Other Kubernetes distributions setup process as well as other custom configurations are referenced in this repository.
+    Switch context to the newly created namespace:
 
-Note: The v1 version of the crd is the one recommended for use and referenced in yaml file names below. However, the v1alpha1 version is the only supported version for running on K8s 1.9 and 1.10. For those versions, use the relevant yamls for v1alpha1. 
+    ```bash
+    kubectl config set-context --current --namespace=demo
+    ```
 
-1. Clone this repo:
-```bash
-git clone https://github.com/RedisLabs/redis-enterprise-k8s-docs
-```
+2. Deploy the operator bundle
 
-2. Create a new namespace:
+   To deploy the default installation with `kubectl`, the following command will deploy a bundle of all the yaml declarations required for the operator:
 
-```bash
-kubectl create namespace demo
-```
+    ```bash
+    kubectl apply -f bundle.yaml
+    ```
 
-Switch context to the newly created namespace:
+    Alternatively, to run each of the declarations of the bundle individually, run the following commands *instead* of the bundle:
 
-```bash
-kubectl config set-context --current --namespace=demo
-```
+    ```bash
+    kubectl apply -f role.yaml
+    kubectl apply -f role_binding.yaml
+    kubectl apply -f service_account.yaml
+    kubectl apply -f crds/app_v1_redisenterprisecluster_crd.yaml
+    kubectl apply -f crds/app_v1alpha1_redisenterprisedatabase_crd.yaml
+    kubectl apply -f operator.yaml
+    ```
+    > Note: The rbac.yaml file used in previous releases has been broken down into three distinct files:
+    `role.yaml`, `role_binding.yaml` and `service_account.yaml`.
+    The `crd.yaml` file was renamed to `redisenterprisecluster_crd.yaml`, with the API version prepended to the filename.
+    Apply the `crds/app_v1alpha1_redisenterprisedatabase_crd.yaml` if managing database instances through Kubernetes API and commands is desired.
```

Comment:
I think that's wrong, this is now part of the bundle. If that is the case, like 72 should be removed. @yuvallevy2

Reply:
This section is for deploying without the bundle though

Reply:
good point there